api-mock/src/main/java/com/dojo/apimock/service/UserDetailsService.java

```diff
@@ -1,22 +1,22 @@
 package com.dojo.apimock.service;
 
-import com.dojo.apimock.ApiMockUserDetailsServiceGrpc;
+import com.codenjoy.dojo.UserDetailsRequest;
+import com.codenjoy.dojo.UserDetailsResponse;
+import com.codenjoy.dojo.UserDetailsServiceGrpc;
 import com.dojo.apimock.LeaderBoardProvider;
-import com.dojo.apimock.UserDetailsRequest;
-import com.dojo.apimock.UserDetailsResponse;
 import io.grpc.stub.StreamObserver;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.LinkedHashMap;
 
 @Service
-public class ApiMockUserDetailsService extends ApiMockUserDetailsServiceGrpc.ApiMockUserDetailsServiceImplBase {
+public class UserDetailsService extends UserDetailsServiceGrpc.UserDetailsServiceImplBase {
```

Comment:
Can service package be checked reordered as it looks a bit messy currently, also there are some classes that are not services